src/daemon/CMakeLists.txt

```diff
@@ -77,7 +77,7 @@ if (MACOS OR LINUX)
               led_keyboard.c
               led_mouse.c
               led_mousepad.c
-              led_wireless.c
+              led_hw.c
```

Comment:
Nitpicking a little bit, but the old name was better. Consider #136 which is *also* a hardware animation system but works very differently.

Reply:
Agreed.